pybossa/view/bloomberg.py

```diff
@@ -75,22 +78,28 @@ def handle_bloomberg_response():
     elif auth.is_authenticated:
         # User is authenticated on BSSO, load user from GIGwork API.
         attributes = auth.get_attributes()
+        current_app.logger.info('User authenitcated but no account found. Attempting to create account for: %s', str(attributes))
```

Comment:
typo here. also can include auth done on bsso.
"User authenticated via bsso but no account found. Attempting to ...."